subrpcserver_config.go

```diff
@@ -41,7 +48,7 @@ type subRPCServerConfigs struct {
 // FetchConfig method.
 func (s *subRPCServerConfigs) PopulateDependencies(cc *chainControl,
 	networkDir string, macService *macaroons.Service,
-	atpl *autopilot.Manager) error {
+	atpl *autopilot.Manager, server *server) error {
```

Comment:
since the server is passed in, we   can remove the separate `chainControl`? Or we could pass in the `chanRouter` directly.